ci-operator/step-registry/ibmcloud/provision/vpc/ibmcloud-provision-vpc-ref.yaml

```diff
@@ -5,6 +5,7 @@ ref:
     name: "4.13"
     tag: upi-installer
   commands: ibmcloud-provision-vpc-commands.sh
+  grace_period: 10m
   resources:
     requests:
       cpu: 10m
```

Comment:
PLs update the step document for `PICKUP_SUBNETS_ORDER` to indicate `2Paging` new option.